mta/entities/__init__.py

```diff
@@ -68,26 +69,40 @@ class AllProjectView(BaseLoggedInPage):
     title = Text(".//div[contains(@class, 'pf-c-content')]/h1")
     search = Input(locator=".//input[@aria-label='Filter by name']")
     sort = SortSelector("class", "btn btn-default dropdown-toggle")
-    projects = ProjectList()
-    new_project_button = Button("Create project")
+
+    ACTIONS_INDEX = 4
+    table = PatternflyTable(
+        ".//table[contains(@class, 'pf-c-table')]",
+        column_widgets={
+            "Name": Text(locator=".//a"),
+            "Applications": Text(locator=".//td[@data-label='Applications']"),
+            "Status": Text(locator=".//td[@data-label='Status']"),
+            "Description": Text(locator=".//td[@data-label='Description']"),
+            ACTIONS_INDEX: Dropdown(),
+        },
+    )
+
+    create_project = Button("Create project")
 
     @View.nested
     class no_matches(View):  # noqa
         """After search if no match found"""
 
-        text = Text(".//div[contains(@class, 'no-matches')]")
-        remove = Text(".//div[contains(@class, 'no-matches')]/a")
+        text = Text(".//div[contains(@class, 'pf-c-empty-state__body')]")
 
     def clear_search(self):
         """Clear search"""
         if self.search.value:
             self.search.fill("")
 
+    @property
+    def all_projects(self):
```

Comment:
I guess this property name can be more generic as "Clear_filters".
Thanks!

Reply:
Nice catch :+1: